### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by irenelagno**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix https://github.com/hlxsites/hsf-commonmoves/issues/12
Fix https://github.com/hlxsites/hsf-commonmoves/issues/43

Test URLs:

Before: https://main--hsf-commonmoves--hlxsites.hlx.page/blog/seller-advice
After: https://blog--hsf-commonmoves--hlxsites.hlx.page/blog/seller-advice?env=stage

Before: https://main--hsf-commonmoves--hlxsites.hlx.page/blog
After: https://blog--hsf-commonmoves--hlxsites.hlx.page/blog?env=stage

After: https://blog--hsf-commonmoves--hlxsites.hlx.page/blog/buyer-advice?env=stage
After: https://blog--hsf-commonmoves--hlxsites.hlx.page/blog/home-improvement?env=stage
After https://blog--hsf-commonmoves--hlxsites.hlx.page/blog/finance?env=stage
After: https://blog--hsf-commonmoves--hlxsites.hlx.page/blog/lifestyle?env=stage
After: https://blog--hsf-commonmoves--hlxsites.hlx.page/blog/general?env=stage

After: https://blog--hsf-commonmoves--hlxsites.hlx.page/blog/blog-detail/2022/09/investment-real-estate-percentage-rules?env=stage

### CODE REVIEW

**Comment by bstopp** [comment]
Can you make this a draft until we get the CORS rules set up?

File: blocks/blog-listing/blog-listing.css [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    .blog-item {
+        flex-basis: 30%;
+    }
+
+    .carousel-list .blog-item {
+        flex-direction: row;
+        flex: 0 0 100%;
+    }
+
+}
```
Comment (AmyJZhao) [inline blocks/blog-listing/blog-listing.css:208]: add newline to end of file

**Comment by AmyJZhao** [comment]
@bstopp
FYI, not sure what's causing this, but some of the blog pages, I'm getting a 404 error
I'm leaning more towards this being an issue on the client's end rather than on our end. maybe their API is including old/deleted pages?
https://blog--hsf-commonmoves--hlxsites.hlx.page/blog/blog-detail/2023/02/sellers---don%E2%80%99t-be-surprised-by-repairs
Screenshot 2023-04-13 at 1 35 24 PM
the commonmoves site: https://www.commonmoves.com/blog/blog-detail/2023/02/sellers---don%E2%80%99t-be-surprised-by-repairs
Screenshot 2023-04-13 at 3 33 03 PM

one more example:
https://blog--hsf-commonmoves--hlxsites.hlx.page/blog/blog-detail/2023/03/small-home-improvements,-big-results

**Review by AmyJZhao — COMMENTED** [review]
Edit: Iryna let me know this is a known issue. Leaving this note here just to document it on the PR:) 
@bstopp
FYI, not sure what's causing this, but some of the blog pages, I'm getting a 404 error
I'm leaning more towards this being an issue on the client's end rather than on our end. maybe their API is including old/deleted pages?
https://blog--hsf-commonmoves--hlxsites.hlx.page/blog/blog-detail/2023/02/sellers---don%E2%80%99t-be-surprised-by-repairs
Screenshot 2023-04-13 at 1 35 24 PM
the commonmoves site: https://www.commonmoves.com/blog/blog-detail/2023/02/sellers---don%E2%80%99t-be-surprised-by-repairs
Screenshot 2023-04-13 at 3 33 03 PM

one more example:
https://blog--hsf-commonmoves--hlxsites.hlx.page/blog/blog-detail/2023/03/small-home-improvements,-big-results

File: blocks/blog-menu/blog-menu.css [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    letter-spacing: 0.1em;
+    padding: 16px 0;
+    border-top: solid 1px var(--border-color);
+}
+
+.blog-nav .blog-nav-selected {
+    text-align: center;
+}
+
+.blog-nav img {
```
[the remark this answers is not in the document]
Comment (irenelagno) [reply blocks/blog-menu/blog-menu.css:84]: good sugestion

**Review by bstopp — APPROVED** [review]

**Review by badvision — APPROVED** [review]
👍🏼

**Pull request merged**